src/qtmozembed/declarativewebpage.cpp

```diff
@@ -68,8 +69,10 @@ DeclarativeWebPage::DeclarativeWebPage(QObject *parent)
     addMessageListener(gLinkAddedMessage);
     addMessageListener(gFindMessage);
     addMessageListener(gContentOrientationChanged);
+    addMessageListener(gOpenLink);
 
     loadFrameScript("chrome://embedlite/content/embedhelper.js");
```

Comment:
This only needed when captive portal browser app is active. I'd not load other wise.

Hence, maybe script should be called something like: "captiveportal.js"